_sass/_py1.scss

```diff
@@ -0,0 +1,156 @@
+// debug
+.py1-wrapper * {
+    // border: solid 1px red;
+}
+
+.py1-wrapper {
+    display: flex;
+    flex-direction: column;
+    margin-top: 10px;
+    p, span, a {
+	// everything seems set in Poppins?
+	font-family: $font-display;
+    }
+    @media (min-width: $desktop-width) {
+	flex-direction: row;
+    }
+}
+
+.illustration img {
+    max-width: 100%;
+}
+
+.py1-sidebar {
+    .py1-illo {
+	width: 60%;
+	img {
+	    max-width: 100%;
+	}
+	@media (max-width: $desktop-width) {
+	    max-width: 40%;
+	}
+    }
+    .py1-fancy-title {
+	line-height: 2em;
+	font-size: 1.6em;
+	font-weight: bold;
+    }
+    .py1-divider-sidebar {
+	visibility: hidden;
+    }
+    .py1-learned-skills-wrapper {
+	p {
+	    margin-bottom: 0;
+	}
+	.py1-learned-skills-list {
+	    margin-top: 0;
+	    padding-left: 1em;
+	    li {
+		padding-left: 0.5em;
+		list-style-type: "\2013";
+	    }
+	}
+    }
+    @media (min-width: $desktop-width) {
+	max-width: 36%;
+    }
+}
+
+// .py1-divider {
+//     // border: none;
+//     margin: 10px 10px;
+//     border-top: 5px solid $teachla-tint;
+//     color: $teachla-tint;
+//     overflow: visible;
+//     height: 100px;
+//     width: 20px;
+// }
+
+// haven't decided on absolutely naming every class
+// or nest classes instead
+// but nesting classes does look chaotic now
```

Comment:
This is a good thing to think about! Keep in mind that the nested classes are compiled into longer CSS declarations. Since CSS evaluates from left-to-right, for `.py1-curriculum section .section-header p`, it'd first:

* look for all `p`'s
* filter out those not in a `.section-header`
* filter out those not in a `section`
* filter those not in the `.py1-curriculum` wrapper

In that sense, you may just want to use a few explicitly-named classes - usually, I don't nest down more than one or two selectors, unless there's a really specific need.

Plus, there's nothing wrong with using long or descriptive names for classes!